pom.xml

```diff
@@ -200,7 +200,7 @@
         <servlet.api.version>2.5</servlet.api.version>
         <slf4j-api.version>2.0.13</slf4j-api.version>
         <slf4j.version>2.0.13</slf4j.version>
-        <solr.version>8.11.2</solr.version>
```

Comment:
Build fails with Solr version updated to 9.4.1.  Solr 8.11.3 is the last supported version in JDK8. Higher versions require JDK11.

Reply:
> Build fails with Solr version updated to 9.4.1. Solr 8.11.3 is the last supported version in JDK8. Higher versions require JDK11.

I noticed that and I couldn't figure out why. In IDE it was working just fine, but not in CLI.